nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-standard-record-utils/src/test/java/org/apache/nifi/schema/validation/TestStandardSchemaValidator.java

```diff
@@ -114,6 +149,100 @@ public void testValidateCorrectSimpleTypesStrictValidation() throws ParseExcepti
         assertTrue(result.getValidationErrors().isEmpty());
     }
 
+    @Test
+    public void testIntegerIsNotFittingForDouble() throws ParseException {
```

Comment:
Some noticeable code duplication across these new tests. Could we do something about it?